src/forms/propertiesdialog.ui

```diff
@@ -673,6 +673,64 @@
                 </item>
                </widget>
               </item>
+              <item row="8" column="0" colspan="3">
+               <layout class="QHBoxLayout" name="horizontalLayout_4">
+                <property name="spacing">
+                 <number>5</number>
+                </property>
+                <item>
+                 <widget class="QLabel" name="fixedSizeLabel">
+                  <property name="text">
+                   <string>Start with this size:</string>
+                  </property>
+                 </widget>
+                </item>
+                <item>
+                 <widget class="QSpinBox" name="fixedWithSpinBox">
+                  <property name="suffix">
+                   <string> px</string>
+                  </property>
+                 </widget>
+                </item>
+                <item>
+                 <widget class="QLabel" name="xLabel">
+                  <property name="text">
+                   <string notr="true">×</string>
+                  </property>
+                 </widget>
+                </item>
+                <item>
+                 <widget class="QSpinBox" name="fixedHeightSpinBox">
+                  <property name="suffix">
+                   <string> px</string>
+                  </property>
+                 </widget>
+                </item>
+                <item>
+                 <widget class="QPushButton" name="fixedSizeButton">
```

Comment:
Might be clearer if this button is named `getCurrentSizeButton`.

Reply:
> Might be clearer if this button is named getCurrentSizeButton.

Please do it by adding a commit to the PR.

> Not sure if the current UI is intuitive or not.

I think it is. An extra "last size" button would be redundant.